yarn.lock

```diff
@@ -7768,9 +7768,13 @@ cz-conventional-changelog@^3.3.0:
   optionalDependencies:
     "@commitlint/load" ">6.1.1"
 
-"czifui@file:czifui-v0.0.21-0.tgz":
-  version "0.0.21-0"
-  resolved "file:czifui-v0.0.21-0.tgz#9edb11babc29b8c0d8dc8e55db107f3f677236b6"
+"czifui@file:czifui-v0.0.23.tgz":
```

Comment:
Just curious, if package.json version is 0.0.24, where did yarn.lock get 0.0.33 from? Do we need this or just 0.0.24 😄 ?

Thanks so much, Masoud!

Reply:
Oh, something must be wrong! 🫠 
I'll check it now.

Reply:
Well, It is fixed 🤓

Reply:
Awesomeeee thanks so much, Masoud 🎉 👏 !